### PR TITLE
Add customization of Wallet position through Mash Settings

### PR DESCRIPTION
**Pull request by mdedys**
## Description

- Added logic to customize position of Wallet based on settings
- Added `background-color` fix to prevent third party styles from setting backgrounds behind the FAB

### CODE REVIEW

File: packages/client-sdk/README.md [hunk 1]
```diff
@@ -49,17 +49,34 @@ ___
 
 Initializes the Mash Wallet. When this method is called, the SDK will load the necessary resources for the Mash Wallet. It will connect the site to the wallet and have it ready for any users that visit your site. 
 
-The function accepts a settings object to help configure the Wallet on your site.
+The function accepts a settings object to help configure the Wallet on your site. Depending on needs of your site, you can shift the Wallet using the `position` key in the settings object. 
+
+You can control on which side the Wallet loads using the `floatLocation` property. If you need further customization, you can shift the Wallet horiontally or vertically based on your needs. There is max horitonal shift of 300px and max vertical shift of 200px.
```
Comment (cartonioh) [inline packages/client-sdk/README.md:54]: typo: `horiontally` and `horitonal`

File: packages/client-sdk/src/settings.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  floatLocation: FloatLocation.BottomRight,
+};
+
+function validateFloatLocation(location: FloatLocation) {
+  return Object.values(FloatLocation).includes(location);
+}
+
+export function merge(settings: MashSettings): MergedMashSettings {
+  // merge settings with default for desktop
+  const desktop = Object.assign(
```
Comment (cartonioh) [inline packages/client-sdk/src/settings.ts:65]: do we need an empty object as first param, before DEFAULT_DESKTOP? wondering if we run the risk of this
Screen Shot 2022-10-26 at 4 20 47 PM

Comment (mdedys) [reply packages/client-sdk/src/settings.ts:65]: What is the issue in this example?

Comment (mdedys) [reply packages/client-sdk/src/settings.ts:65]: Yah I can do that.

**Review by nyonson — APPROVED** [review]

**Pull request merged**